File: service/app/src/test/java/io/miragon/blueprint/architecture/ArchitectureTest.java
package io.miragon.blueprint.architecture;

import io.miragon.common.architecture.ServiceArchitectureTest;

/**
 * Runs the reusable ArchUnit suite against this service's base package.
 * The rules live in the {@code common-architecture-tests} module.
 */
class ArchitectureTest extends ServiceArchitectureTest {

    @Override
    protected String rootPackage() {
        return "io.miragon.blueprint";
    }
}

File: service/app/src/test/kotlin/io/miragon/blueprint/architecture/ArchitectureTest.kt
package io.miragon.blueprint.architecture

import io.miragon.common.architecture.ServiceArchitectureTest

/**
 * Runs the reusable ArchUnit + Konsist suite against this service's base package.
 * The rules live in the `:service:common-architecture-tests` module.
 */
class ArchitectureTest : ServiceArchitectureTest("io.miragon.blueprint")
